src/Console/ReplayCommand.php

```diff
@@ -35,7 +36,7 @@ public function handle(Projectionist $projectionist): void
 
     public function selectProjectors(array $projectorClassNames): ?Collection
     {
-        if (count($projectorClassNames ?? []) === 0) {
+        if (count($projectorClassNames ?? []) === 0 && ! $this->option('force')) {
```

Comment:
Instead of using `&&` just add the check above the existing code.

Reply:
Sure thing, that’ll be way more readable. Will post a fix later today.

Reply:
Sorry about the delay @freekmurze 

Here is what I had in mind for a refactor of my first suggestion:

```php
public function selectProjectors(array $projectorClassNames): ?Collection
    {
        if (count($projectorClassNames ?? []) === 0) {

            if ($this->option('force')) {
                return $this->projectionist->getProjectors();
            }

            if (! $confirmed = $this->confirm('Are you sure you want to replay events to all projectors?', true)) {
                return null;
            }

            return $this->projectionist->getProjectors();

        }

  // ...
```

To prevent a bunch of commits here, I wanted to hear if this seems like a good approach.

I have to admit, this is mye very first ever PR to an open source project (not counting documentation patches), so I am trying to thread carefully here. 
I also see that you prefer to have multiple commits in a PR squashed before submitting. I have never tried that before, but I could give it a try here if it is possible for a PR that has already been submitted.

Reply:
Don't worry about too many commits, I'll squash them all myself before merging 👍 

Thanks for contributing!

Instead of checking the `force` flag yourself, use Laravels `ConfirmatbleTrait` which checks this for your. Here an example of how you can use it: https://github.com/spatie/laravel-db-snapshots/blob/5c1b58da6d1898eb5d2a4b31d6287d27b86f8188/src/Commands/Load.php#L13